docs/source/reproducers/03-zuul.md

```diff
@@ -0,0 +1,155 @@
+# Zuul job
+## Words of warning
+This feature is still fresh, and limited. For now, you can reproduce only
+a job that doesn't rely on the content-provider job.
+
+It also has to run against the latest PR content output, since you won't
+have access to the actual commit hash on github.
+
+For now, it has been tested only against
+*podified-multinode-edpm-e2e-nobuild-tagged-crc* job.
+
+## Words of advice
```

Comment:
Min reqs may deserve their own section at the top. Or it could be linked from elsewhere in the docs, provided it's already defined.

Reply:
min  reqs values change with the intended use. For the plain layout, we may need smaller resources. For that zuul reproducer? it may require more, due to tempest being launched at the end for instance, or just in case of 3-compute layout (for ceph HCI - it's coming).
It's hard to properly size things - that's also why the doc links to CRC specs (in the 01-considerations).